GiniVision/Assets/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,117 @@
+/* 
+  Localizable.strings
+  GiniVision
+
+  Created by Peter Pult on 08/06/16.
+  Copyright © 2016 Gini. All rights reserved.
+*/
+
+"ginivision.navigationbar.camera.title"        = "Photograph document";
+"ginivision.navigationbar.onboarding.title"    = "Instructions";
+"ginivision.navigationbar.review.title"        = "Check photo quality";
+"ginivision.navigationbar.analysis.title"      = "Etwas Geduld, analysiere Foto";
+"ginivision.navigationbar.camera.close"        = "Close";
+"ginivision.navigationbar.camera.help"         = "Help";
+"ginivision.navigationbar.onboarding.continue" = "Next";
+"ginivision.navigationbar.review.back"         = "Back";
+"ginivision.navigationbar.review.close"        = "Close";
+"ginivision.navigationbar.review.continue"     = "Next";
+"ginivision.navigationbar.analysis.back"       = "Cancel";
+"ginivision.navigationbar.help.backToCamera"   = "Back";
+"ginivision.navigationbar.help.backToMenu"     = "Back";
+
+
+"ginivision.camera.captureButton" = "Capture";
+"ginivision.camera.notAuthorized" = "Permission to access the camera is required to analyse photos";
+"ginivision.camera.notAuthorizedButton" = "Grant permission";
+"ginivision.camera.captureFailed" = "Image capture failed";
+"ginivision.camera.filepicker.photoLibraryAccessDenied" = "Permission to access your photos gallery is required to analyse saved photos";
+"ginivision.camera.filepicker.mixedDocumentsUnsupported" = "It is only possible to select a file type at the same time.";
+"ginivision.camera.filepicker.errorPopup.cancelButton" = "Cancel";
+"ginivision.camera.filepicker.errorPopup.confirmButton" = "OK";
+"ginivision.camera.filepicker.errorPopup.grantAccessButton" = "Grant permission";
+"ginivision.camera.documentValidationError.general" = "Unfortunately, this file can not be edited";
+"ginivision.camera.documentValidationError.excedeedFileSize" = "Unfortunately, this file is too big";
+"ginivision.camera.documentValidationError.tooManyPages" = "The document can only have a maximum of 10 pages.";
+"ginivision.camera.documentValidationError.wrongFormat" = "Only files of the formats jpg, png, gif or pdf can be analysed";
+"ginivision.camera.fileImportTip" = "Now documents can be conveniently uploaded for analysis.";
+"ginivision.camera.fileImportButtonLabel" = "Import";
+"ginivision.camera.capturedImagesStackLabel" = "Erfasste Dokumente";
+"ginivision.camera.qrCodeDetectedPopup.message" = "Use QR code?";
+"ginivision.camera.qrCodeDetectedPopup.buttonTitle" = "Continue";
+"ginivision.camera.mixedarrayspopup.cancel" = "Cancel";
+"ginivision.camera.mixedarrayspopup.usePhotos" = "Use photos";
+"ginivision.camera.errorPopup.cancelButton" = "Cancel";
+"ginivision.camera.errorPopup.pickanotherfileButton" = "Pick another file";
```

Comment:
`pickanotherfileButton` is not camel case.

Reply:
I just checked that it is customizable so we cannot change it unfortunately. 😢

Reply:
Maybe we can add the correct one and deprecate this one? Something, something, future

Reply:
That would make it more confusing, since there isn't a formal way to deprecate a string. We can keep it as it is.